### ISSUE TITLE
error BadRequest with special character &

### ISSUE THREAD

**Issue by Noah-Rod** [pkkid/python-plexapi#483]
Hello 
With the new PlexApi version (3.4). Before I had the version 3.1
Everithing runs much faster: A huge Compliment to the Developers.
But I run into an error. When I want to get information about a series and there is an "&" in the Name. With the Version 3.1 I don't have that issue

Following is my Code:
`server = PlexServer(spez.PlexUrl, spez.PlexToken)`
`library = server.library.section(spez.PlexSeriesLibrary)`
`library.get("Serie Title with &")`

Raise following Error:
`plexapi.exceptions.BadRequest: (400) bad_request`
P.s. the series titel i get through a for loop, so it's not a typo

I hope somebody can help.

**Comment by Hellowlol** [comment]
Use .all or search

**Comment by Noah-Rod** [comment]
Yes I said it before i am iterating over the series here is it how i did it.
```
series = library.search()
    for ser in series:
        progress()
        seasons = library.get(ser.title).seasons()
```

**Comment by Hellowlol** [comment]
Really depends on what your trying to do. But if you want iterate over all shows in all libraries you can do

````python
series = library.search(libtype="show")
for show in series:
    seasons = show.seasons()
````

**Comment by Noah-Rod** [comment]
Your solution with `show.seasons()` worked Thank you for that. I used alawys the other method `libary.get(show.title).seasons()` because the last time I programmed with the api i ran in to an Error after searching what is wrong. I found that it is recommeded to use my method. 
But now your method works. Thank you for your answer. 
Should I close the bug? For me I can work with your method. I would say the other Method should also be possible, to not to run in an error.
Again Thank you a lot for your help.

**Comment by Noah-Rod** [comment]
Hello I have also an complete different question. Can you say me how i find out if a scan from the media libary is running or the trash gets deleted. How could I track that?
I want to make ssure nothing is rnning when i make a backup. Iknow the session method bt that gives me only users who are streaming.

**Comment by Hellowlol** [comment]
You need to use the alert listener and check for event that indicate this.
https://python-plexapi.readthedocs.io/en/latest/modules/alert.html#plexapi.alert.AlertListener

**Comment by amazingr4b** [comment]
I noticed this today when trying to watch law & order. I added url = url.replace(" & "," %26 ") to the def query function in the server.py file to prevent that & from causing the data after it from being seen as another variable.

**Comment by Hellowlol** [comment]
Fixed with 709dabac5bf349f7721bdf8e23e011c371cf5fb4
